### PR TITLE
Added info about signing debug variant

### PR DESCRIPTION
**Pull request by ftmzhrsckldr**

### CODE REVIEW

File: docs/workflows/android-specific-workflow-steps/android-sign.md [hunk 1]
```diff
@@ -16,6 +16,16 @@ This step follows the [**Android Build**](/workflows/android-specific-workflow-s
 
 :::
 
+:::caution Debug Variant Signing  
+
+As noted in the [Android Developer documentation](https://developer.android.com/build/build-for-release):
```
Comment (boztopuz) [inline docs/workflows/android-specific-workflow-steps/android-sign.md:21]: I think, it is an emphasis. IMO, we should make it bold.

```suggestion
As noted in the [**Android Developer documentation**](https://developer.android.com/build/build-for-release):
```

File: docs/workflows/android-specific-workflow-steps/android-sign.md [hunk 2]
```diff
@@ -16,6 +16,16 @@ This step follows the [**Android Build**](/workflows/android-specific-workflow-s
...
+
+> If the build variant you've selected is a debug build type, then the APK is signed with a debug key and it's ready to install. If you've selected a release variant, then, by default, the APK is unsigned and you must manually [sign the APK](https://developer.android.com/studio/publish/app-signing).
+
+When you build your app using the **debug** variant and select a keystore in [configurations](/build/platform-build-guides/building-android-applications#signing), the **Android Sign** step will replace the default debug signing files and re-sign the app using the specified keystore files.
```
Comment (boztopuz) [inline docs/workflows/android-specific-workflow-steps/android-sign.md:25]: Sometimes, users dont understand what they should do in configraution. IMO we should specify the correct address for specific screens.

```suggestion
When you build your app using the **debug** variant and select a keystore from the signing tab in [configurations](/build/platform-build-guides/building-android-applications#signing), the **Android Sign** step will replace the default debug signing files and re-sign the app using the specified keystore files.
```

File: docs/workflows/android-specific-workflow-steps/app-post-processor.md [hunk 3]
```diff
@@ -7,11 +7,22 @@ tags: [android, mobile, post processor]
...
 :::warning
 
-This step also verifies whether the app is signed or not. If this step is not included in your Workflow or if it is determined that there is no signed app as a result of this step, __the app cannot be distributed__.
+This step also verifies whether the app is signed or not. If this step is not included in your workflow or if it is determined that there is no signed app as a result of this step, **the app cannot be distributed**.
+
+:::
+
+:::info Debug Variant Signing  
+
+As noted in the [Android Developer documentation](https://developer.android.com/build/build-for-release):
```
Comment (boztopuz) [inline docs/workflows/android-specific-workflow-steps/app-post-processor.md:21]: Same suggestion here.
```suggestion
As noted in the [**Android Developer documentation**](https://developer.android.com/build/build-for-release):
```

File: docs/workflows/android-specific-workflow-steps/app-post-processor.md [hunk 4]
```diff
@@ -7,11 +7,22 @@ tags: [android, mobile, post processor]
...
+
+> If the build variant you've selected is a debug build type, then the APK is signed with a debug key and it's ready to install. If you've selected a release variant, then, by default, the APK is unsigned and you must manually [sign the APK](https://developer.android.com/studio/publish/app-signing).
+
+This means that when you build your app with the **debug** variant, the **Android Post Processor** step in Appcircle will recognize the app as already signed, even if you did not sign it in Appcircle or your repository.
```
Comment (boztopuz) [inline docs/workflows/android-specific-workflow-steps/app-post-processor.md:25]: Grammer correction suggested. ->

```suggestion
This means that when you build your app using the **debug** variant, the **Android Post Processor** step in Appcircle will recognize the app as already signed, even if it was not signed in Appcircle or your repository.
```

**Review by boztopuz — CHANGES_REQUESTED** [review]

File: docs/workflows/android-specific-workflow-steps/app-post-processor.md (hunk 4 above)
Comment (ftmzhrsckldr) [reply docs/workflows/android-specific-workflow-steps/app-post-processor.md:25]: IMO "with" would be more correct (I updated the other document like this). I applied the other feedback.

File: docs/workflows/android-specific-workflow-steps/android-sign.md (hunk 2 above)
Comment (ftmzhrsckldr) [reply docs/workflows/android-specific-workflow-steps/android-sign.md:25]: The link I provided shows which tab of the Configuration they need to open, but a little extra information never hurts! :)
I applied

**Review by boztopuz — APPROVED** [review]

**Pull request merged**